src/Data/Textile/Economics.elm

```diff
@@ -0,0 +1,182 @@
+module Data.Textile.Economics exposing
+    ( Economics
+    , Price
+    , computeDurabilityIndex
+    , computeMarketingDurationIndex
+    , computeNumberOfReferencesIndex
+    , computeRepairCostIndex
+    , decode
+    , decodePrice
+    , maxMarketingDuration
+    , maxNumberOfReferences
+    , maxPrice
+    , minMarketingDuration
+    , minNumberOfReferences
+    , minPrice
+    , priceFromFloat
+    , priceToFloat
+    )
+
+import Data.Unit as Unit
+import Duration exposing (Duration)
+import Json.Decode as Decode exposing (Decoder)
+import Json.Decode.Pipeline as Pipe
+
+
+type alias Economics =
+    { marketingDuration : Duration
+    , numberOfReferences : Int
+    , price : Price
+    , repairCost : Price
+    }
+
+
+{-| Note: We don't want to specify a specific currency (eg. EUR, USD)
+so we can keep prices abstract yet consistent.
+TODO: move to Unit?
```

Comment:
Note: I'm really not sure about that as I really liek the idea of it to be tied to the `Economics` module. Thoughts?